frontend/src/components/forms/InviteEmployeesModal.tsx

```diff
@@ -0,0 +1,314 @@
+            mb="10px"
+            ml="10px"
+            mr="10px"
+            borderBottom="1px solid #EBECF0"
+          >
+            <ModalHeader>Add Employee</ModalHeader>
+            <ModalCloseButton mt="5px" margin="inherit" size="lg" />
+          </Box>
+          <ModalBody>
+            <Box pt="7px" pb="7px" pl="16px" pr="16px">
+              <Box
+                display="flex"
+                flexDirection="row"
+                alignItems="flex-start"
+                justifyContent="space-between"
+              >
+                <Box w="46%">
+                  <FormControl isRequired isInvalid={invitedFirstNameError}>
+                    <FormLabel>First Name</FormLabel>
+                    <Input
+                      placeholder="Enter first name"
+                      value={invitedFirstName}
+                      onChange={handleFirstNameChange}
+                      maxLength={50}
+                    />
+                    <FormErrorMessage>First Name is required.</FormErrorMessage>
+                  </FormControl>
+                </Box>
+
+                <Box w="46%">
+                  <FormControl isRequired isInvalid={invitedLastNameError}>
+                    <FormLabel>Last Name</FormLabel>
+                    <Input
+                      placeholder="Enter last name"
+                      value={invitedLastName}
+                      onChange={handleLastNameChange}
+                      maxLength={50}
+                    />
+                    <FormErrorMessage>Last Name is required.</FormErrorMessage>
+                  </FormControl>
+                </Box>
+              </Box>
+              <Box display="flex" flexDirection="row" mt="16px">
+                <FormControl isRequired isInvalid={invitedEmailError}>
+                  <FormLabel>SHOW Email</FormLabel>
+                  <Input
+                    placeholder="Enter SHOW email"
+                    value={invitedEmail}
+                    onChange={handleEmailChange}
+                    maxLength={254}
+                  />
+                  <FormErrorMessage>A SHOW email is required.</FormErrorMessage>
+                </FormControl>
+              </Box>
+            </Box>
+            <Box
+              pt="17px"
+              pb="7px"
+              pl="24px"
+              pr="24px"
+              borderBottom="1px solid #EBECF0"
+            >
+              <FormControl isRequired isInvalid={invitedAdminStatusError}>
+                <RadioGroup
+                  value={invitedAdminStatus}
+                  onChange={handleAdminChange}
+                  pb="24px"
+                >
+                  <Flex>
+                    <Box marginRight="24px">
+                      <Radio
+                        size="lg"
+                        colorScheme="teal"
+                        color="black"
+                        value="1"
+                      >
+                        Admin
```

Comment:
I would wrap the text in a Text component, so it gets the default styling 
```suggestion
                        <Text>Admin</Text>
 ```